docs/tools/draco.md

```diff
@@ -0,0 +1,84 @@
+# Draco: a Chemical Data Extractor
+
+[Draco] is a tool which aims to extract chemical data (Molecules, Reactions, procedures, etc.) from documents. In its first version, Draco focuses at extracting molecules (fragments and complete structures) from PDF documents
```

Comment:
we have to say that this is decimer

Reply:
No, we can rebrand it as it's on MIT License. However, we should include the license in the GitHub repo related to Draco. As the code is not exposed to the customers, I don't think it's really matter, but I am not a lawyer, so I am not sure. At the moment, we are using Decimer and Decimer-segmentation. The last one is going to be changed. The masks are either going to be merged with the one from YOLO, or YOLO is going to replace it.

Reply:
But, yes we can specify and provide more details regarding the workflow and the tools involved